code/modules/mob/living/carbon/human/species.dm

```diff
@@ -898,7 +925,8 @@ GLOBAL_LIST_EMPTY(roundstart_races)
 		if(!I.species_exception || !is_type_in_list(src, I.species_exception))
 			return FALSE
 	if(I.species_restricted & H.dna?.species.bodyflag)
-		to_chat(H, "<span class='warning'>Your species cannot wear this item!</span>")
+		if(!disable_warning)//Wow im a dipshit
```

Comment:
Why is this comment there?

Reply:
Also logic is backwards

Reply:
Alot of these comments / code isn't mine, so I can't give insights but, I'm more than willing to fix anything.

Reply:
![image](https://user-images.githubusercontent.com/40559528/185055473-e98dc910-5152-4951-9b09-b600f352429d.png)